packages/evm/examples/clz.ts

```diff
@@ -0,0 +1,61 @@
+import { Common, Hardfork, Mainnet } from '@ethereumjs/common'
+import { type EVM, createEVM } from '@ethereumjs/evm'
+import { type PrefixedHexString, hexToBytes } from '@ethereumjs/util'
+
+const common = new Common({
```

Comment:
Could we add a little bit more description/context as to what this does and what the purpose of this example is? We could also link to the EIP https://eips.ethereum.org/EIPS/eip-7939 etc.